includes/utils.php

```diff
@@ -23,6 +23,7 @@ class Utils {
 	 * A list of safe tage for `validate_html_tag` method.
 	 */
 	const ALLOWED_HTML_WRAPPER_TAGS = [
+		'a',
```

Comment:
@matipojo 
Who else is using this const?
I hope it will not allow a tags, form elements/ other clickable elements nested under another a tag
Any thoughts?

Reply:
It just validates is an HTML tag